.travis.yml

```diff
@@ -12,7 +17,7 @@ before_script:
 script:
 - |
   # get ourselves a MIR-ful libstd
-  xargo/build.sh
+  cd xargo && RUSTFLAGS='-Zalways-encode-mir' xargo build && cd ..
```

Comment:
the portability issues are `readlink`. A quick google search tells me that I need to use `greadlink` after a call to `brew install coreutils`. Which doesn't help portability ;)